lib/util.js

```diff
@@ -4,6 +4,7 @@ const config = require('./config')
 const fs = require('fs-extra')
 
 const runGClient = (args, options = {}) => {
+  if (options.verbose) args.push('--verbose')
```

Comment:
we might as well just access `config.gClientVerbose` directly here. If any other options are passed to `gclientSync` or `gclientRunhooks` then the verbose option will be lost the way it's used now